src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -97,6 +103,18 @@
 
     BookReader.prototype.initMenuToggle = function brInitMenuToggle(e) {
       var br = this;
+      let hasNav = false;
```

Comment:
QUESTION: Does this need to be ES5 compatible, i.e. use var instead of let?

Reply:
No; that's what the build step is for.